main.js

```diff
@@ -567,6 +569,16 @@ const initializeDictationConnection = async () => {
         }
         if (typeof mes.data === "string") {
             const data = JSON.parse(mes.data);
+
+            if (data.type === "AUDIO_CHUNK_ACK") {
+                if (data.ack_id <= lastAckId) {
```

Comment:
ours servers guarantee that this is never the case, it will be confusing to have the check in the sample app. I'd suggest putting just a comment to say "this is where you'd remove audio chunks from your buffer"